images/scripts/lib/atomic.install

```diff
@@ -168,7 +168,7 @@ class AtomicCockpitInstaller:
         """ Install the latest cockpit RPMs in our container"""
         rpm_args = []
         for package in self.rpms:
-            if 'cockpit-ws' in package or 'cockpit-dashboard' in package or 'cockpit-bridge' in package:
+            if 'cockpit-ws' in package or 'cockpit-bridge' in package:
```

Comment:
rhel-atomic is also RHEL 7, let's not touch this.

Reply:
rhel-atomic or atomic? (you commented on atomic)

Reply:
rhel-atomic.install is the only remaining OS install script that uses this file. So they are really synonyms these days.